chain/network/src/routing.rs

```diff
@@ -442,9 +448,13 @@ impl RoutingTable {
             store,
             raw_graph: Graph::new(peer_id),
             route_nonce: SizedCache::with_size(ROUND_ROBIN_NONCE_CACHE_SIZE),
+            #[cfg(feature = "adversarial")]
             ping_info: SizedCache::with_size(PING_PONG_CACHE_SIZE),
+            #[cfg(feature = "adversarial")]
             pong_info: SizedCache::with_size(PING_PONG_CACHE_SIZE),
+            #[cfg(feature = "adversarial")]
             waiting_pong: SizedCache::with_size(PING_PONG_CACHE_SIZE),
+            #[cfg(feature = "adversarial")]
```

Comment:
It might be better if we create a single struct to hold all of these fields for testing

Reply:
I agree

Reply:
Done